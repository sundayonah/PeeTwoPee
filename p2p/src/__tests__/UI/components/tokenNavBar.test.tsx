import * as React from 'react';
import { MockedProvider } from '@apollo/client/testing';
import { render, RenderResult,screen } from '@testing-library/react';
import TokenNavBarHeader from "../../../pages/Sell/components/TokenNavbarHeader"


describe('<TokenNavBar />',()=>{
    it("renders out a default token",()=>{
        render(<TokenNavBarHeader/>)
        const BNBToken = screen.getByText("BNB")
        expect(BNBToken).toBeInTheDocument()
    })
    it("Matches snapshot",()=>{
        const {container} = render(<TokenNavBarHeader />)
        expect(container).toMatchSnapshot()
    })
})