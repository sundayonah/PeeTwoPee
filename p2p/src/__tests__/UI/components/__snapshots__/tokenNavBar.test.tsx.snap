// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TokenNavBar /> Matches snapshot 1`] = `
<div>
  <div
    class="css-46j4ko"
  >
    <div
      class="css-1iqq0oq"
    >
      RGP
    </div>
    <div
      class="css-1iqq0oq"
    >
      BUSD
    </div>
    <div
      class="css-1iqq0oq"
    >
      USDT
    </div>
    <div
      class="css-1iqq0oq"
    >
      BNB
    </div>
    <div
      class="css-1hkcd3u"
    >
       Add Custom Token  
      <svg
        class="chakra-icon css-onkibi"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
          fill="currentColor"
        />
      </svg>
    </div>
  </div>
</div>
`;
